app/src/main/java/com/infomaniak/mail/utils/FetchMessagesManager.kt

```diff
@@ -249,5 +249,10 @@ class FetchMessagesManager @Inject constructor(
 
     companion object {
         private val TAG: String = FetchMessagesManager::class.java.simpleName
+
+        // Based on what seems to be the limit taken into account by NotificationCompat.Builder, we truncate some fields to avoid
+        // pendingIntent that will trigger TransactionTooLargeException further down the line like sentry id 41593. We don't need
+        // the extra data, it was bound to be truncated anyway
```

Comment:
```suggestion
        // Based on what seems to be the limit taken into account by NotificationCompat.Builder, we truncate some fields to avoid
        // pendingIntent that will trigger TransactionTooLargeException further down the line like Sentry ID 41593. We don't need
        // the extra data, it was bound to be truncated anyway.
```